DiplomaTests/DiplomaTests/Tests/AuthenticationTests/LoginTest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using DiplomaTests.Utility;
+
+namespace DiplomaTests.Tests
+{
+
+    [TestFixture(BrowserType.Chrome)]
+    public class LoginTest : TestBase
+    {
+        public LoginTest(BrowserType browser) : base(browser)
+        {
+        }
+
+        [Test , Order(1)]
+        public void ValidUserCanLogin()
+        {
+            LoginAsValidUser();
+        }
+
+        [Test, Order(2)]
+        public void InvalidUserCannotLogin()
+        {
+            loginPage.EnterUsername("aaaa");
+            loginPage.EnterPassword("bbbb");
+            loginPage.ClickLoginButton();
+            loginPage.GetInvalidLoginMessage();
```

Comment:
where is assert 
![image](https://github.com/user-attachments/assets/626a2708-08bf-4a51-85be-b0bf3ac23847)